docs/opcodes/FD.mdx

```diff
@@ -5,18 +5,30 @@ group: System operations
 
 ## Notes
 
-Equivalent to [CREATE](/#F0), except the salt allows to create at a deterministic address.
+Abort execution, undo all state changes, return a value and refund the remaining gas to the caller.
```

Comment:
Thanks, but this is not complete :-)
FD should be renamed to F5 first, and then FD can take those contents.

Reply:
That's nice but not complete :-)
I would say "Stop the current context execution, revert the state changes (see [STATICCALL](/#FA) for a list of state changing opcodes) and return unused gas to the caller. If the execution is stopped with REVERT, the value `0` is put on the stack of the calling context, which continues to execute normally".

"Refund" has a specific meaning, let's not use it here, it's not the same thing.